### DISCUSSION TITLE
How to organize and showcase multiple projects in my GitHub profile README without it looking messy?

### DISCUSSION THREAD

**Discussion by AshuraMajestic** [community/community#138775] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone,

I'm looking to enhance my GitHub profile by showcasing multiple projects in my README.md. However, I want to present them in a clean and organized way, avoiding clutter. What are some effective strategies or formatting techniques to display project details (like links, descriptions, or badges) while keeping the profile looking professional and easy to navigate?

Any examples or tips would be greatly appreciated!

Thanks in advance!

**Comment by SuGAr9807** [comment, marked as answer]
I have given a site where i showed all my projects using a slider.
